core/arch/arm/tee/arch_svc.c

```diff
@@ -124,7 +124,8 @@ static void trace_syscall(size_t num)
 {
 	if (num == TEE_SCN_RETURN || num > TEE_SCN_MAX)
 		return;
-	FMSG("syscall #%zu (%s)", num, tee_svc_syscall_table[num].name);
+	FMSG("syscall #%zu (%s)", num,
+	     tee_svc_syscall_table[SPECULATION_SAFE_VALUE(num)].name);
```

Comment:
Doesn't really hurt, but since this is a debug print that shall be disabled on production devices it doesn't add that much to use the macro here.

Reply:
Agree, I added it here for completeness.
It is a classic spectre pattern so it good to tag it as such.